shared/wallets/send-form/asset-input/asset-input-basic-container.tsx

```diff
@@ -13,7 +13,6 @@ const mapStateToProps = state => {
     currencyLoading: currency === '',
     displayUnit: Constants.getCurrencyAndSymbol(state, currency) || currency,
     // TODO differentiate between an asset (7 digits) and a display currency (2 digits) below
-    inputPlaceholder: Constants.inputPlaceholderForCurrency(currency),
```

Comment:
I think sep7-confirm was using this, it has its own asset-input container, maybe needs to change its props